src/operator/numpy/np_elemwise_broadcast_op.cc

```diff
@@ -186,6 +212,21 @@ MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_npi_power_scalar)
 .set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Compute<cpu, mshadow_op::power>)
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseIn{"_backward_power_scalar"});
 
+MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_npi_logaddexp_scalar)
+.set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Compute<cpu, mshadow_op::logaddexp>)
+.set_attr<nnvm::FGradient>("Fgradient", ElemwiseGradUseIn{"_backward_logaddexp_scalar"});
+
+
+MXNET_OPERATOR_REGISTER_BINARY(_backward_logaddexp_scalar)
+.add_argument("scalar", "float", "scalar value")
+.set_attr_parser([](NodeAttrs *attrs) { attrs->parsed = std::stod(attrs->dict["scalar"]); })
+.set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Backward<
+  cpu, mshadow_op::logadd_left>);
+
+
+
+
+
```

Comment:
too many blank lines here

Reply:
blank lines removed, thanks for reviewing

Reply:
> too many blank lines here

blank lines removed, thanks for reviewing